packages/core/chrome/core-chrome-browser-internal/src/project_navigation/project_navigation_service.test.ts

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { firstValueFrom } from 'rxjs';
+import { ProjectNavigationService } from './project_navigation_service';
+import { applicationServiceMock } from '@kbn/core-application-browser-mocks';
+import type { ChromeNavLinks } from '@kbn/core-chrome-browser';
+
+const setup = () => {
+  const projectNavigationService = new ProjectNavigationService();
+  const projectNavigation = projectNavigationService.start({
+    application: applicationServiceMock.createInternalStartContract(),
+    navLinks: {} as unknown as ChromeNavLinks,
+  });
+
+  return { projectNavigation };
+};
+
+describe('breadcrumbs', () => {
+  test('should return list of breadcrumbs home / nav / custom', async () => {
```

Comment:
Looks like the 2 tests in the suite are duplicated. Maybe the top one is not needed for now?